### ISSUE TITLE
Unable to read disc (my solutions)

### ISSUE THREAD

**Issue by Dec1234** [CTurt/FreeDVDBoot#108]
Saw a few others leaving their experiences with how they got this working when they ran into issues.

I have a Slim PS2, with **3.10E** and a version 9300 system.

For my first attempt i tried Intenso brand DVD+RW discs. I tried everything I could but could not even get any of these discs to boot. Eventually after some trial and error I managed to get both the 3.1/3.11 hybrid disc to get to a "Unable to read disc" blackscreen, which was at least an improvement over having nothing. This only happend after decided to use InfraRecorder instead of ImgBurn, and I also tested a DVD movie first, and removed all my memory cards/controllers as other users suggested.

Next I purchased some Verbatim DVD-R discs and tried again, but hit the same problem. It was only after seeing some lost forum post from 2008 with someone mentioning how to get a backup copy of a disc to read, and it was only after using CDBurnerXP could he get it to work.

You can probably now guess what my solution was. ImgBurn and InfraRecorder (both with default + adjusted write settings) will not allow any discs to work for me. CDBurnerXP worked instantly. Disc was burned with 1x speed and loads instantly with zero issues. I have even gone and tested this again to ensure I didnt make a mistake but no, same repeatable problems for me when using the other tools to burn the disc.

Hope this helps someone else.

**TLDR:** _CDBurnerXP Portable x64 with DVD Finalize + Compatibility setting enabled, worked fine for me with DVD-R discs. ImgBurn and InfraRecorder would not work no matter what settings I tried._

Also great work CTurt on figuring this out and putting it altogether.

**Comment by TnA-Plastic** [comment]
I would have turned OPC in IMGBurn's write-settings on and 1x speed? What kind of burner is that? It must have a low laser-power.

**Comment by CTurt** [comment]
Thanks, will update README.
